dashboard/src/index.tsx

```diff
@@ -7,19 +7,10 @@ import { axios } from "./shared/Auth";
 import Root from "./containers/Root";
 import "./index.css";
 import store from "./store";
-// import registerServiceWorker from "./registerServiceWorker";
 
 createAxiosInterceptors(axios, store);
 
 ReactDOM.render(<Root />, document.getElementById("root") as HTMLElement);
 
-// Disable the service worker as it currently loads index.html from cache for
-// all unknown paths. This doesn't work well with our Ingress setup because if
-// you visit /kubeless/ you end up getting the Dashboard instead of the Kubeapps
-// UI. We can consider re-enabling this once the Kubeless UI is integrated in
-// the Dashboard and we no longer need the external link.
-
-// registerServiceWorker();
```

Comment:
@andresmgot It's called here, but indeed the comment is old and since we no longer have a different app under `/kubeless`, we may want to consider enabling this. I would keep this commented out for now, and update the comment to a TODO to look into re-enabling the service worker.